_data/MeshanKhosla/MeshanKhosla.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Meshan Khosla  
+institution: UC Berkeley  
+profile_pic: meshan.jpg  
+quote: "If High School was filmed like The Office, I'd be Kevin dropping the chili everyday"  
```

Comment:
Please remove the quotes to maintain the syntax format

Reply:
Done. Do I need to submit another form?

Reply:
If you have already submitted a form, then no need to submit another one